klippy/extras/print_stats.py

```diff
@@ -73,6 +78,11 @@ def get_status(self, eventtime):
             'state': self.state,
             'message': self.error_message
         }
-
+    def cmd_PAUSE_STATS(self,gcmd):
+        if self.last_pause_time is None:
+            curtime = self.reactor.monotonic()
+            self.last_pause_time = curtime
```

Comment:
Filament usage should be updated via a call to `self.update_filament_usage()` here.  This would make sure the filament usage is up to date before tracking stops.